apps/web/components/Comparison/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ElementType, MouseEventHandler, ReactNode } from 'react'
+import { FocusableBox, Link, Text, Box } from '@island.is/island-ui/core'
+import * as styles from './Comparison.css'
+
+type ColorScheme = 'blue' | 'red'
+
+interface ComparisonProps {
+  children?: ReactNode
+  colorScheme?: ColorScheme
+  href?: string
+  onClick?: MouseEventHandler
+}
+
+const colorSchemes = {
+  blue: {
+    backgroundColor: 'blue100',
+    textColor: 'blue400',
+    tagVariant: 'darkerBlue',
+    bordered: false,
+  },
+  red: {
+    backgroundColor: 'red100',
+    textColor: 'red600',
+    tagVariant: 'red',
+    bordered: true,
+  },
+} as const
+
+export const Comparison: React.FC<React.PropsWithChildren<ComparisonProps>> = ({
```

Comment:
This component doesn't look like it will be reused throughout the web project, perhaps keeping it in a folder close to where its used instead would be ideal?

Reply:
allright, I have moved it :)